x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -465,4 +521,15 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       './actions'
     );
   }
+
+  private lazyAssistantSettingsManagement() {
```

Comment:
<img width="619" alt="image" src="https://github.com/elastic/kibana/assets/2946766/ec410926-4be8-4243-a5db-25323cb89f49">

👏👏👏 Thank-you!